#   Here's how to use it:
#   1. In File Explorer, locate the file /home/sws/tlr_ws/src/tlr_detector/hsv_tuner.html.
#   2. Open this file in a web browser (e.g., Chrome, Firefox).
#   3. Once the webpage loads, click the “Red”, ‘Yellow’, or “Green” button at the top to select the color you want to adjust.
#   4. Move the sliders to adjust the Hue, Saturation, and Value. The color preview box and the text below will change in real-time.
#   5. After finding the optimal HSV values using this tool, you can directly modify and save those values in the config/tlr_hsv_params.yaml file.
/**:
  ros__parameters:
    # --- YELLOW ---
    yellow_h_min: 20
    yellow_h_max: 40
    yellow_s_min: 100
    yellow_s_max: 255
    yellow_v_min: 100
    yellow_v_max: 255

    # --- GREEN ---
    green_h_min: 50
    green_h_max: 80
    green_s_min: 100
    green_s_max: 255
    green_v_min: 100
    green_v_max: 255

    # --- RED ---
    red_h_min1: 0
    red_h_max1: 10
    red_s_min1: 100
    red_s_max1: 255
    red_v_min1: 100
    red_v_max1: 255

    red_h_min2: 160
    red_h_max2: 180
    red_s_min2: 100
    red_s_max2: 255
    red_v_min2: 100
    red_v_max2: 255
